frontend/src/components/worksheets/ExtraWorksheetHTML/ExtraWorksheetHTML.scss

```diff
@@ -31,4 +31,10 @@
     padding: 5px 6px 8px 6px;
     border-color: #eee;
     border-bottom: 1px solid #eee;
-  }
\ No newline at end of file
+  }
+#glossaryModal {
```

Comment:
The position looks weird...also the shortcut '?' is broken saying
`TypeError: jquery__WEBPACK_IMPORTED_MODULE_8___default(...)(...).modal is not a function
`
![Screenshot from 2019-11-13 10-07-56](https://user-images.githubusercontent.com/23012631/68791231-ba60eb00-05fd-11ea-8705-67e0f76e4022.png)

Reply:
The position should be working fine. You should edit `index.js` so that it live reloads the css from `overrides.scss`.

Reply:
The jquery modal is another issue present on the `frontend` branch too. Making another issue for it.

Reply:
ah yes, it's working fine now, but the shortcut is still broken after the refactor commit